File: day 35 python/homework/hw2.py
# 3) ფუნქცია იღებს სიას რიცხვებისგან და აბრუნებს მხოლოდ დადებითების ჯამს.

def sum_of_positive(nums):
    result = []
    for i in nums:
        if i>0:
            result.append(i)
    return sum(result)

print(sum_of_positive([1,2,3,4,5,   -1,-2,-3,-4,-5]))

File: day 55 Js/homework/hw2.js
// დაწერეთ კოდი, რომელიც აიღებს მასივს [10,15,20,25], თუ რიცხვი მეტია 18-ზე, გამოიტანოს "yes", სხვა შემთხვევაში "no"

let nums = [10,15,20,25];

for (let i = 0; i < nums.length; i++){
    if (nums[i] > 18){
        console.log("yes")
    } else{
        console.log("no")
    }

}